types/node-tlv/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictFunctionTypes": true,
+        "strictNullChecks": true,
+        "baseUrl": "../",
+        "typeRoots": [
+            "../"
+        ],
+        "types": [],
+        "noEmit": true,
+        "forceConsistentCasingInFileNames": true,
+        "esModuleInterop": true
```

Comment:
```suggestion
        "forceConsistentCasingInFileNames": true
```

Reply:
there is no real reason to use it here for DT, it's CJS module:
https://github.com/coolbong/node-tlv/blob/master/lib/TLV.js#L1067
It has sense for consumer tsconfig though.